public/app/features/dashboard/components/DashboardSettings/GeneralSettings.test.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { byRole, byText } from 'testing-library-selector';
```

Comment:
This is an interesting helper library, I'll have to check it out. Ah wait it's @domasx2 lib, awesome!